docs/sdks/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
+
+As of 2023-07-15 plugin version:
+* iOS V 0.2.0
+* Android V 0.4.0
```

Comment:
@genwhittTTD adding this because the versions for the GMA plugins have already diverged to different numbers. 

Also we should add a date to show this release note was updated at that date and the description is good for the versions above